### DISCUSSION TITLE
Fatal Error install

### DISCUSSION THREAD

**Discussion by Joostvab** [liedekef/events-made-easy#459] [Q&A] [answered]
When installing Versie 2.4.25 we get the following error when we try to activate!

Fatal error: Cannot redeclare eme_plugin_row_meta() (previously declared in /home/u7524p5126/domains/noord-nederlandse-vliegvis-vereniging.nl/public_html/wp-content/plugins/OUDevents-made-easy/events-manager.php:168) in /home/u7524p5126/domains/noord-nederlandse-vliegvis-vereniging.nl/public_html/wp-content/plugins/events-made-easy/events-manager.php on line 173

The old plugin is also visibel but Versie 2.3.14 But both don't work anymore

What to do.....

**Comment by liedekef** [comment]
Don't keep 2 versions of a plugin in the WP plugin folder, wordpress will include them both.
So remove the OUDevents-made-easy folder and try again. Also: we're at 2.4.29 ...

**Comment by Joostvab** [comment]
If you remove the old Event made easy plugin are the reservation data  keep or are they also be removed??

**Reply by liedekef** [reply, marked as answer]
Just update EME as specified in the readme, all data should remain present. As always (and recommended by WP): make a db backup before updating.
